gradle-plugin/src/main/groovy/akka/grpc/gradle/AkkaGrpcPlugin.groovy

```diff
@@ -140,7 +141,9 @@ class AkkaGrpcPlugin implements Plugin<Project>, DependencyResolutionListener {
     @Override
     void beforeResolve(ResolvableDependencies resolvableDependencies) {
         def compileDeps = project.getConfigurations().getByName("compile").getDependencies()
-        compileDeps.add(project.getDependencies().create("com.lightbend.akka.grpc:akka-grpc-runtime_2.12:${pluginVersion}"))
+        boolean foundRuntimeDep = compileDeps.any { dep -> dep.group == "com.lightbend.akka.grpc" && dep.group.startsWith("akka-grpc-runtime") && dep.version == pluginVersion }
```

Comment:
I wonder if we could find the `scala-library` dependency in a similar way and add the right runtime based on that?

Reply:
Yeah. On my way to the office I was thinking maybe adding an argument to the plugin so things are a bit less magic. 

I'm a bit unsure about how the scala version is set on a gradle build. Unlike `sbt` in the [gradle plugin for scala](https://docs.gradle.org/current/userguide/scala_plugin.html) there doesn't seem to be a need (or even a mechanism?) to explicitly set the desired scala version. There is a recommendation to add the dependency to `'org.scala-lang:scala-library:2.x.y'` but I have a pet project without the explicit dep that compiles and runs tests successfully.